src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -616,3 +617,37 @@
                                  (icons/icon {} (if (:collapsed? @state) :expand :collapse)))
           body]
          body)))})
+
+(react/defc QueueStatus
```

Comment:
We're trying to move away from the monolithic `components.cljs`. Let's put this in its own file in the `components` package. Then since it'll be the only thing in it, you can `:refer` it in the imports.